cuda_core/tests/test_utils.py

```diff
@@ -13,11 +13,37 @@
 import cuda.core.experimental
 import numpy as np
 import pytest
-from cuda.core.experimental import Device
+from cuda.bindings import driver
+from cuda.core.experimental import Buffer, Device, MemoryResource
 from cuda.core.experimental._memoryview import view_as_cai
+from cuda.core.experimental._utils.cuda_utils import handle_return
 from cuda.core.experimental.utils import StridedMemoryView, args_viewable_as_strided_memory
 
 
+class DummyDeviceMemoryResource(MemoryResource):
```

Comment:
Not a huge fan of this duplication, I can move this to the helpers library if desired.